XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -262,6 +262,31 @@ source code for the XAML Controls Gallery. -->
             </local:ControlExample.Substitutions>
         </local:ControlExample>
 
+        <local:ControlExample HeaderText="ListView with Filtering"
+                              XamlSource="ListView/ListViewSample4_xaml.txt"
+                              CSharpSource="ListView/ListViewSample4_cs.txt">
+            <ListView 
+                x:Name="FilteredListView"
+                ItemTemplate="{StaticResource ContactListViewTemplate}"
+                SelectionMode="Single"
+                ShowsScrollingPlaceholders="True"
+                Height="400"
+                Width="400"
+                BorderThickness="1"
+                BorderBrush="{ThemeResource SystemControlForegroundBaseMediumLowBrush}">
```

Comment:
nit: please close the ListView inline instead of having `</ListView>` hang out on its own line.